### DISCUSSION TITLE
Implementing control in a form

### DISCUSSION THREAD

**Discussion by victorsolaya** [OOlashyn/PCF-AttachmentGalleryControl#23] [Q&A] [answered]
Hi @OOlashyn 
Thank you for the control. 

At the moment we want to implement this, however, there is no luck as Controls doesn't appear in the Timeline and when we want to add Notes as a subgrid there is no annotations.
How should we proceed with this? 
Probably this would be beneficial to be in the documentation README 😊

Thank you!

**Comment by OOlashyn** [comment, marked as answer]
Hi @victorsolaya,
I added an installation process to the Wiki page, so please check it for further details. My control isn't placed on top of timeline - you need to place it on top of String type field. If you will have any other questions after checking Wiki feel free to reach out back to me.

**Reply by victorsolaya** [reply]
Hi @OOlashyn  many thanks for the quick reply! As I looked to the ControlManifest and I saw it was a standard control I wasn't sure how to put this on.
But yes, now it makes sense 😊😊 Thanks for the quick help!
